sematic/log_reader.py

```diff
@@ -16,17 +17,30 @@
 from sematic.scheduling.external_job import JobType
 from sematic.storage import S3Storage
 
-# Why the "V1"? Because we will likely want to change the structure of
-# the logs such that each file contains a different subset of logs. But
-# when we make this change, we will still want logs written in the old
+# Why the "V1"/"V2"? Because we changed the structure of the logs. Originally,
+# each file on s3 held the entirety of the logs. Now each file contains a
+# different subset of logs. We still want logs written in the old
 # structure to be readable, at least for a while. So we need to identify
 # which structure the files are in somehow, and a v1/v2 prefix is how we
 # can do it.
-V1_LOG_PATH_FORMAT = "logs/v1/run_id/{run_id}/{log_kind}/"
+V1_LOG_PREFIX = "logs/v1"
+V2_LOG_PREFIX = "logs/v2"
```

Comment:
The old format *also* has multiple files, it's just that in that case, each file is a snapshot of the logs at a different point in time. Trying to read from those in the same way as logs are read from in the new format would lead to repeating a bunch of logs.

Reply:
Can this be cleanup at a later time?

Reply:
> Can this be cleanup at a later time?

Yes, probably a few releases from now. I just want to provide some continuity in ability to read things